pkg/dataflatten/flatten_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sort"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
The automated tools I used to stub out the tests uses assert, I'll see if I can change it to require. 

However, in given that it takes a few min for the CI to complete, assert might be superior. If I understand it correctly ....

If a test that uses assert fails, you will get a log for each assertion that failed.

If a test that uses require fails, you will only get a log for the first require that failed since it interrupts the test.

Reply:
Yes, that's the difference.

But in something like a mini sub-test, if the err isn't what's expected, there's no point in continuing the test. I think about it something like:
```go
require.NoError(t, err)
assert.Equal(t, val1, expected1)
assert.Equal(t, val2, expected2)
```